contracts/staking/StakingStorage.sol

```diff
@@ -82,3 +82,7 @@ contract StakingV1Storage is Managed {
     // Allowed AssetHolders: assetHolder => is allowed
     mapping(address => bool) public assetHolders;
 }
+
+contract StakingV2Storage is StakingV1Storage {
+    mapping(address => address) public rewardsDestination;
```

Comment:
We could add a comment here to describe what `rewardsDestination` is

Reply:
Good idea